vision/unit_tests/test_blobbing.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+import os
+import sys
+import cv2
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
+
+try:
+    from vision.blob.blobfind import BlobFinder
+except ImportError:
+    from blob.blobfind import BlobFinder
+
+class TestBlobbing(unittest.TestCase):
+    def test_finding_blobs(self):
+        """
+        Tests that the expected number of blobs is found
+
+        Settings
+        --------
+        expected_blobs: list[int]
+            expected blobs to be found in each image
+
+        Returns
+        -------
+        list[bool]
+            whether the expected number of blobs in each image equals the detected number of blobs
+        """
+        expected_blobs = {
+            "apple.jpg": 1,
+            "legos.jpg": 30,
+            "MyBeach.png": 1,
+            "oranges.png": 1,
+            "sampleobj.png": 1
+        }
+        vision_folder = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+        for i, img in enumerate(os.listdir('vision/blob/samples')):
```

Comment:
We can refactor this to 
```
for filename, expected in expected_blobs.items():
```
I think that will clean up your code below and prevent it from breaking with new files in the samples directory.